### DISCUSSION TITLE
Mode expert > Vérifications DNS

### DISCUSSION THREAD

**Discussion by tamere-allo-peter** [lschaeffer313/domaine-nc-mobile#111] [Ideas]
* Conformité DNSSec : Je ne sais pas s'ils proposent une API mais cet outil est super utile et ce serait sympa soit de l'utiliser, soit de reproduire son fonctionnement : https://dnssec-analyzer.verisignlabs.com/
* Conformité DNS : mêmes remarques que ci-dessus : https://mxtoolbox.com/SuperTool.aspx?action=dns

**Comment by adriens** [comment]
Yes, généralement ils demandent un compte les gars ;-(
